### PR TITLE
Import playlist from json file

### PR DESCRIPTION
**Pull request by HaiDang666**

### CODE REVIEW

File: packages/app/app/actions/playlists.js [hunk 1]
```diff
@@ -70,3 +72,51 @@ export function updatePlaylist(playlist) {
...
+
+        if (_.isEmpty(tracks)) {
+          dispatch(error('Playlist import fail', 'Empty data', null, null));
+          dispatch({
+            type: null
+          });
+          return;
+        }
+
+        if (playlists) {
```
Comment (nukeop) [inline packages/app/app/actions/playlists.js:104]: just do `playlists = [...playlists, playlist]` here

File: packages/app/app/components/Playlists/PlaylistsHeader/index.tsx [hunk 2]
```diff
@@ -1,18 +1,33 @@
...
 
 type PlaylistsHeaderProps = {
   showText: boolean;
 }
 
-const PlaylistsHeader: React.FC<PlaylistsHeaderProps> = ({showText}) => {
+const PlaylistsHeader: React.FC<PlaylistsHeaderProps> = ({ showText }) => {
   const { t } = useTranslation('playlists');
+  const dispatch = useDispatch();
+  const handleImportFromFile = useCallback(async () => {
```
Comment (nukeop) [inline packages/app/app/components/Playlists/PlaylistsHeader/index.tsx:19]: Can you move this callback to the playlists container?

**Comment by HaiDang666** [comment]
updated changes

File: packages/app/app/actions/playlists.js [hunk 3]
```diff
@@ -70,3 +68,47 @@ export function updatePlaylist(playlist) {
     });
   };
 }
+
+export function addPlaylistFromFile(filePath) {
+  return async dispatch => {
+    fs.readFile(filePath, (err, data) => {
+      if (err) {
+        dispatch(error('Playlist import fail', 'Can not open file', null, null));
```
Comment (nukeop) [inline packages/app/app/actions/playlists.js:76]: Just remembered this: these message strings need to be localized and defined in `en.json`

Comment (HaiDang666) [reply packages/app/app/actions/playlists.js:76]: I'll try to pass `t` from `useTranslation` from the container

Comment (nukeop) [reply packages/app/app/actions/playlists.js:76]: Ok, that's the best way to do it. Or just pass an object with message strings as an argument.

**Pull request merged**